lib/stripe/account.rb

```diff
@@ -13,7 +13,8 @@ def url
     end
 
     # @override To make id optional
-    def self.retrieve(id=nil, opts={})
+    def self.retrieve(id=ARGUMENT_NOT_PROVIDED, opts={})
+      id = id.equal?(ARGUMENT_NOT_PROVIDED) ? nil : Util.check_api_key!(id)
```

Comment:
`id` can be an account ID, so while the string check is correct the error message about api_keys would be confusing here.